src/cachemachine/rubinrepoman.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, List, Set
+
+import structlog
+from docker_registry_client import DockerRegistryClient
+
+from cachemachine.dockercreds import DockerCreds
+
+logger = structlog.get_logger(__name__)
+
+
+class RubinRepoMan:
+    registry_url: str
+    recommended_image_url: str
+    recommended_tag: str
+    num_dailies: int
+    num_weeklies: int
+    num_releases: int
```

Comment:
Same note here about class variables.

Reply:
Sorry I though this is just the types of the instance variables since I don't have an =?  Maybe I'm just completely confused about this typing stuff.

Reply:
Actually, scratch that, I _am_ confused by this typing!